src/data/option/basic.lean

```diff
@@ -179,11 +181,20 @@ by { cases x; simp only [map_none, map_some, eq_self_iff_true] }
   x.map f = none ↔ x = none :=
 by { cases x; simp only [map_none', map_some', eq_self_iff_true] }
 
+/-- `option.map` is an injective function. -/
+theorem map_injective' : function.injective (@option.map α β) :=
+λ f g h, funext $ λ x, some_injective _ $ by simp only [← map_some', h]
```

Comment:
How is this different from the unprimed version?

Reply:
The unprimed version says that `option.map f` is injective whenever `f` is injective.

Reply:
I think it's fine as is, I just misread the docstring